### PR TITLE
Create vc2017x86run.txt

### PR DESCRIPTION
**Pull request by blackcrack**
for make modern Programs to run
with vb2017-x86

### CODE REVIEW

**Comment by blackcrack** [comment]
Hi Alex,
the x86 pack installs well and it is in my testpack. 
Reactos-test-packs.iso (01-forwinerosextralibrarys57.exe in it ) it's an
library and dll installer and this vc2017x86.exe it is install pretty well without errors
on x86_Reactos Trunks/Daily's, the vc2017x86_64.exe on ROS64 should works well too imho..
Some programs like notepad3 works with it.

**Comment by sanchaez** [comment]
All right @blackcrack. The general commit policy still applies here - I only need your real name to merge 😊

**Comment by blackcrack** [comment]
*lol* how long be i am at the Ros-Forum and by the whole Project ...
greetings to Jaix 👍  ask him, who is the founder of Rapps ..

File: vc2017x86run.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ Description = Needed by applications compiled with Visual Studio 2017.
 Size = 14,56 MiB
 Category = 14
 URLSite = https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads
+URLSite1 = https://github.com/WPN-XM/vcredist
```
Comment (HBelusca) [inline vc2017x86run.txt]: What that is for? There is nothing but a readme at this URL.

File: vc2017x86run.txt [hunk 2]
```diff
@@ -0,0 +1,65 @@
+[Section]
+Name = Microsoft Visual C++ 2017 Redistributable(x86)
```
Comment (HBelusca) [inline vc2017x86run.txt]: may need a space before "(x86")

Comment (blackcrack) [reply vc2017x86run.txt]: 👍 letz do it .. :)

File: vc2017x86run.txt (hunk 1 above)
Comment (blackcrack) [reply vc2017x86run.txt]: the possible for add 1,2 other url's if give a Webspace , github and maybe a Forumspage .
therefore should we have a possible for ulrside ,1,2,3.

Also would it be great if we have a "unattended=" or "swiches =" possible
best regards
Blacky

**Comment by blackcrack** [comment]
for this, Program announce/updating !
that is nonsense and has nothing to do with programming !
Programming in Reactos, yes, indeed, right, to know from where and who has programming, but there ?!!
nope ! You trust me or you trust me not ! I be now more as 10 Years on the side of Reactos !

**Pull request closed without merging**